src/components/CandidateContest.tsx

```diff
@@ -275,25 +275,16 @@ class CandidateContest extends React.Component<Props, State> {
   }
 
   public closeAttemptedVoteAlert = () => {
-    // Delay to avoid passing tap to next screen
-    window.setTimeout(() => {
-      this.setState({ attemptedOvervoteCandidate: undefined })
-    }, 200)
```

Comment:
This required removing these delays similarly to what I did in #966. I couldn't reproduce the issue that these timeouts were trying to fix.